common/php/class-module.php

```diff
@@ -253,7 +253,7 @@ function enqueue_datepicker_resources() {
 	function get_current_post_type() {
 		global $post, $typenow, $pagenow, $current_screen;
 		//get_post() needs a variable
-		$post_int;
+		$post_int = '';
```

Comment:
Can probably simplify to this:

```
$post_int = isset( $_REQUEST['post'] ) ? (int)$_REQUEST['post'] : false;
```

In any case, we need to change one of the `elseif` blocks below:

```
&& isset( $_REQUEST['post'] )
 			&& isset( $post_int )
```

to

```
&& $post_int
```